v2/pkg/genruntime/conversion_helpers.go

```diff
@@ -105,3 +107,19 @@ func GetOptionalIntValue(ptr *int) int {
 func GetIntFromFloat(f float64) int {
 	return int(math.Round(f))
 }
+
+// ConvertJSONToString returns the string value of the given v1.JSON.
+func ConvertJSONToString(json v1.JSON) string {
+	if json.Raw == nil {
+		return ""
+	}
+	
+	return string(json.Raw)
+}
+
+// ConvertStringToJSON returns the v1.JSON value of the given string.
+func ConvertStringToJSON(s string) v1.JSON {
+	return v1.JSON{
+		Raw: []byte(s),
```

Comment:
Is there no factory function for `v1.JSON` values?

Reply:
Done!